src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -448,8 +445,7 @@ internal void InsertGraph()
         private string DynamoGraphFromMDFilePath(string path)
         {
             var dynPath = Path.Combine(Path.GetDirectoryName(path), Path.GetFileNameWithoutExtension(path).Replace("%20", " ")) + ".dyn";
-            return dynPath;
-            return path.Replace("%20", " ").Replace(".md", ".dyn");
+            return dynPath.Replace("%20", " ");
```

Comment:
I think it is more robust to use a url decode method https://learn.microsoft.com/en-us/dotnet/api/system.web.httputility.urldecode?source=recommendations&view=net-7.0

Reply:
@mjkkirschner even if the DynamoGraphFromMDFilePath method is called from 2 different places I will need to use the UrlDecode method is both places, is that ok?

Reply:
Fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/13863/commits/5c3666153466e7fa04b950918fd7f7b46664a79f